vtm/src/org/oscim/map/Animator.java

```diff
@@ -193,6 +193,8 @@ public void animateFling(float velocityX, float velocityY,
     }
 
     private void animStart(float duration, int state) {
+        if (!isActive())
```

Comment:
Why the need of the check here?
From my tests the method is called 1 time per animation.